src/main/converter/opds.ts

```diff
@@ -153,13 +155,20 @@ export class OpdsFeedViewConverter {
         let type = OpdsResultType.Empty;
         let navigation: OpdsLinkView[] | undefined;
         let publications: OpdsPublicationView[] | undefined;
+        let nextPageUrl: string;
+        let previousPageUrl: string;
 
         if (feed.Publications) {
             // result page containing publications
             type = OpdsResultType.PublicationFeed;
             publications = feed.Publications.map((item) => {
                 return this.convertOpdsPublicationToView(item);
             });
+
+            if (feed.Links) {
+                nextPageUrl = oc(feed.Links.find((link) => link.Rel[0] === "next")).Href(undefined);
+                previousPageUrl = oc(feed.Links.find((link) => link.Rel[0] === "previous")).Href(undefined);
```

Comment:
Strictly-speaking in Atom XML `link@rel` is optional (even though in practice OPDS links always seem to provide the link relation), so I would check `link.Rel` for non-undefined before accessing the array `link.Rel[0]`.
https://tools.ietf.org/html/rfc4287#section-4.2.7

Reply:
An OPDS2 link can carry multiple relations, such as `previous` and `first` (or `last` and `next`) in no particular order. So I would not only check for `link.Rel[0]`, but I would instead implement a lookup routine such as:
`=> link.Rel && link.Rel.includes("previous")`
https://drafts.opds.io/opds-2.0#3-pagination